src/client/app/components/DateRangeComponent.tsx

```diff
@@ -2,63 +2,52 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import * as React from 'react';
-import { useEffect, useState } from 'react';
-import { useSelector, useDispatch } from 'react-redux'
 import DateRangePicker from '@wojtekmaj/react-daterange-picker';
-import { CloseReason, Value } from '@wojtekmaj/react-daterange-picker/dist/cjs/shared/types';
-import 'react-calendar/dist/Calendar.css';
 import '@wojtekmaj/react-daterange-picker/dist/DateRangePicker.css';
+import { Value } from '@wojtekmaj/react-daterange-picker/dist/cjs/shared/types';
+import * as React from 'react';
+import 'react-calendar/dist/Calendar.css';
+import { useDispatch } from 'react-redux';
+import { selectQueryTimeInterval, updateTimeInterval } from '../redux/slices/graphSlice';
+import { useAppSelector } from '../redux/reduxHooks';
+import { Dispatch } from '../types/redux/actions';
 import { dateRangeToTimeInterval, timeIntervalToDateRange } from '../utils/dateRangeCompatibility';
-import TooltipMarkerComponent from './TooltipMarkerComponent';
 import translate from '../utils/translate';
-import { State } from '../types/redux/state';
-import { ChartTypes } from '../types/redux/graph';
-import { Dispatch } from '../types/redux/actions';
-import { changeGraphZoomIfNeeded } from '../actions/graph';
+import TooltipMarkerComponent from './TooltipMarkerComponent';
+import { selectSelectedLanguage } from '../redux/slices/appStateSlice';
+
+// Potential Fixes, for now omitted
```

Comment:
Could you explain what the fix is?

Reply:
There's an issue with the date range picker overflowing & getting overshadowed by the plotly graph when the screen size is not wide enough.

 By default the date picker does not honor the width of the parents container. Enabling this behavior with those few lines of CSS fixes this issue, but I'm not too keen on how it looks visually.

Reply:
I added a small comment to document.